### DISCUSSION TITLE
Confusion about vue 2.7 support

### DISCUSSION THREAD

**Discussion by MartinX3** [vuejs/vue-cli#7223] [Q&A]
According to the releases 5.0.6 enabled support for vue 2.7, but 2.6.14 is still used
https://github.com/vuejs/vue-cli/blob/4a0655f7ac09b64d2b47506e7f21e7923d43262b/packages/%40vue/cli/package.json#L60

That gives me the funny error message I need to workaround by enforcing the installation via `npm install --force`

Error:
```
npm ERR!   vue@"^2.7.2" from the root project
npm ERR!   vue@"^2.6.14" from @vue/cli@5.0.7
```

**Comment by Viggo95** [comment]
`^2.6.14` means that when installing dependencies it will look for the **latest minor version**


So the real version in the lock file will be 2.7

**Reply by MartinX3** [reply]
And the error?
